vint/_bundles/__init__.py

```diff
@@ -0,0 +1,6 @@
+from __future__ import absolute_import
+
+try:
+    import vimlparser
+except ImportError:
+    from . import vimlparser
```

Comment:
Could you add a comment that explains "We can use the system-wide vimlparser module when it found.", please?